tests/plugins/test_add_labels.py

```diff
@@ -295,3 +295,34 @@ def test_add_labels_aliases(tmpdir, docker_tasker, caplog,
 
     if expected_log:
         assert expected_log in caplog.text()
+
+def test_dont_overwrite_distribution_scope(tmpdir, docker_tasker):
+    df_content = "FROM fedora\n"
+    df_content += 'LABEL distribution-scope="private"'
+    labels_conf_base = {INSPECT_CONFIG: {"Labels": {"distribution-scope": "private"}}}
+
+    df = df_parser(str(tmpdir))
+    df.content = df_content
+
+    if MOCK:
+        mock_docker()
+
+    workflow = DockerBuildWorkflow(MOCK_SOURCE, 'test-image')
+    setattr(workflow, 'builder', X)
+    flexmock(workflow, base_image_inspect=labels_conf_base)
+    setattr(workflow.builder, 'df_path', df.dockerfile_path)
+
+    runner = PreBuildPluginsRunner(
+        docker_tasker,
+        workflow,
+        [{
+            'name': AddLabelsPlugin.key,
+            'args': {'labels': {"distribution-scope": "restricted"}, "dont_overwrite_if_in_dockerfile": ["distribution-scope"], "auto_labels": [],
```

Comment:
I wonder if we should test that this is the default behaviour, i.e. not provide a `dont_overwrite_if_in_dockerfile` parameter, as well.